### PR TITLE
Fix spelling and typos

### PR DESCRIPTION
**Pull request by jonaski**
I started testing out tagparser with Strawberry (https://github.com/strawberrymusicplayer/strawberry) and I found some typos and spellings while reading the sources to figure it out.

### CODE REVIEW

File: mp4/mp4track.cpp [hunk 1]
```diff
@@ -452,7 +452,7 @@ TrackHeaderInfo Mp4Track::verifyPresentTrackHeader() const
  * \brief Reads the sample to chunk table.
  * \returns Returns a vector with the table entries wrapped using the tuple container. The first value
  *          is an integer that gives the first chunk that share the same samples count and sample description index.
- *          The second value is sample cound and the third value the sample description index.
+ *          The second value is sample counted and the third value is the sample description index.
```
Comment (Martchus) [inline mp4/mp4track.cpp:455]:
```suggestion
 *          The second value is sample count and the third value is the sample description index.
```

**Review by Martchus — COMMENTED** [review]
Thanks, it look good except for one change. Did you use a tool for spell checking the source code?

**Comment by jonaski** [comment]
I did some manually, and then I can codespell.

**Comment by Martchus** [comment]
I guess I'll need to do the same on my other projects as well.

**Pull request merged**